Add procedure to select columns in the host table (#1928)
theforeman/foreman-documentation: guides/common/modules/proc_selecting-host-columns.adoc

[id="selecting-host-columns_{context}"]
= Selecting Host Columns

You can select what columns you want to see in the host table on the *Hosts* > *All Hosts* page.
For a complete list of host columns, see xref:overview-of-the-host-columns_{context}[]

> [NOTE]

Reviewer: ```suggestion

[NOTE]
```

====

> It is not possible to deselect the *Name* column. The *Name* column serves as  a primary identification method of the host.

Reviewer: ```suggestion
It is not possible to deselect the *Name* column.
The *Name* column serves as a primary identification method of the host.
```

====

.Procedure

> . In the Satellite web UI, navigate to *Hosts* > *All Hosts*.

Reviewer: ```suggestion
. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
```

[id="selecting-host-columns_{context}"]
= Selecting Host Columns

You can select what columns you want to see in the host table on the *Hosts* > *All Hosts* page.

> For a complete list of host columns, see xref:overview-of-the-host-columns_{context}[]

Reviewer: ```suggestion
For a complete list of host columns, see xref:overview-of-the-host-columns_{context}[].
```
